### ISSUE TITLE
gen_req ignores `--req_email=...` when also using `--batch`

### ISSUE THREAD

**Issue by eengstrom** [OpenVPN/easy-rsa#184]
Perhaps intentional, but it seems that `easyrsa3` ignores global options such as `--req_email=..` when running with `--batch` enables.  If not intentional, seems like a relatively easy fix, but before I attempt any fix, is this intended behavior?

As a workaround, when calling `easyrsa`, I am currently utilizing the environment to pass in a different email, as in:

      (export EASYRSA_REQ_EMAIL="${EMAIL_TO}";
       easyrsa --batch  gen-req $CRT_NAME
     )

**Comment by ecrist** [comment]
Can you confirm this is still happening after https://github.com/OpenVPN/easy-rsa/commit/9bd3241a31e019b1f41d63bda31a929af1502da7?

**Comment by eengstrom** [comment]
So, I tested with that version and the error does not exist.  I then returned to my own sandbox with an older version and retested, and it works as expected.  I revisited my own wrapper script which uses this functionality and found the error **in my code**, so it's not an error with `easyrsa`.  

However, I did discover that unrecognized options or commands result in warnings, not errors.  The source of the error is that I originally tried (stupidly) to put the `--req-email=` option after the command, as in:

    easyrsa --batch gen-req TEST --req-email="test@nowhere.com"

Instead, I needed to put the options together before the `gen-req` command.  

There was a warning issued, but the warning was lost to me because of the wrapper script which would have exited on an error (`set -e`), but no non-zero exit code meant that I never saw the warning.  Still my problem, but perhaps you might consider more aggressive options when unrecognized options or commands are provided.

Bottom line, this is a false report.  Sorry to bother you.  Thanks for the attention and time on the software.
